src/openfl/_internal/text/TextEngine.hx

```diff
@@ -899,35 +901,40 @@ class TextEngine {
 
 					case JUSTIFY:
 						if (lineWidths[lineIndex] < totalWidth) {
-							lineLength = 1;
-
+							spacesInLine = 1;
+							layoutGroupsInLine = 1;
 							for (j in (i + 1)...layoutGroups.length) {
 								if (layoutGroups[j].lineIndex == lineIndex) {
 									if (j == 0 || text.charCodeAt(layoutGroups[j].startIndex - 1) == " ".code) {
```

Comment:
i starts from 0 and j starts from i + 1. Therefore j can never be 0 so j==0 check is obsolete as will always be false